### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by josefie**
Hi, I added a little style to the page. Please let me know if you have any suggestions for improvement.

### CODE REVIEW

**Comment by justmarkup** [comment]
Awesome, thanks a lot for this.

**Comment by marcus-herrmann** [comment]
It's really great!

Just some non-dealbreaking suggestions:
* Adding an indicator for external links
* Border-Radius for buttons
* Usage of `aria-current="page"` on main nav

**Comment by justmarkup** [comment]
@marcus-herrmann 

- [x] added `aria-current="page"` for the main nav

I don't really get the point about border-radius?
And by indicator you mean an icon or/and a text explaining that an external site will open? I think the link text are all pretty clear that an external site will open there and we also don't use `target="_blank"`, but curious to hear what you had in mind?

**Comment by marcus-herrmann** [comment]
Thank you for adding aria-current! By external link indicator I meant something like this: https://codepen.io/simevidas/pen/MKdmeV, but you're right - it's clear within context.

Regarding button border radius - it's for an easier recognition as a button, as Maja mentioned in Wednesday during our pre-meeting. I read that somewhere too, will search it in my bookmarks...

File: about.html [hunk 1]
```diff
@@ -1,5 +1,26 @@
 --- 
 layout: default
 title: About a11y Berlin
-description: a11y Berlin is an usergroup about accessibility and inclusive design.
+description: a11y Berlin is a usergroup about accessibility and inclusive design.
```
Comment (radibit) [inline about.html]: Minor content suggestion – could we please change `a11y` with first letter capitalized everywhere where we have it as a part of the meet-up name (including the `<title>`) 👉 `A11y Berlin`?

File: about.html [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
 ---
+
+<p>
+  A11y Meetup Berlin is designed to bring together the accessibility and inclusive design community to share and discuss ideas, best practices, and experiences. 
+  We would love to invite everyone interested in these topics.
+</p>
+
+<aside>
+  <h2>What does "a11y" mean?</h2>
```
Comment (radibit) [inline about.html:13]: It's really nice that we have the explanation of the acronym, but maybe we should also have a short explanation on what web accessibility means e.g. 

> ...the degree to which a website is accessible by as many people as possible

– Laura Kalbag

**Review by radibit — APPROVED** [review]
Looks really nice 🎉  Great job.

I left just two minor comments regarding the content.

File: about.html (hunk 1 above)
Comment (justmarkup) [reply about.html]: Changed it.

File: about.html (hunk 2 above)
Comment (justmarkup) [reply about.html:13]: @radibit Opened an issue for this.

**Pull request merged**